library/scss/src/settings/components/_checkbox.scss

```diff
@@ -165,6 +165,9 @@ $checkbox-image: 80px;
 
 					.forminator-checkbox-image {
 						box-shadow: 0 3px 8px 0 rgba(0,0,0,0.3);
+						@include media(max-width, md) {
+							box-shadow: 0 1px 1px 0 rgb(0 0 0 / 30%);
```

Comment:
@creador-dev, on front libraries, never use RGB + slash for alpha colors because it is a new kind of annotation not supported by old IE browsers, and we still support IE 9+. Please, consider using RGBA instead.